tests/Fixtures/AbstractResetPasswordTestKernel.php

```diff
@@ -90,7 +90,7 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
                     'App' => [
                         'is_bundle' => false,
                         'type' => 'annotation',
-                        'dir' => '%kernel.project_dir%/tests/Fixtures/Entity/',
+                        'dir' => 'tests/Fixtures/Entity/',
```

Comment:
why changing that to a relative path ?

Reply:
@stof https://github.com/jrushlow/reset-password-bundle/runs/574232421

```
1) SymfonyCasts\Bundle\ResetPassword\Tests\FunctionalTests\Command\ResetPasswordRemoveExpiredCommandTest::testRemoveExpiredCommandWithNothingToRemove
InvalidArgumentException: Specified non-existing directory "/home/runner/work/reset-password-bundle/reset-password-bundle/tests/FunctionalTests/tests/Fixtures/Entity/" as Doctrine mapping source.
```

Is this a bug in master or did I foul something up on my end?

Reply:
Well, the issue is that MicroKernelTrait defines `getProjectDir` defines the project dir as the parent dir of the location of the kernel class. But here, this abtract class is *not* the kernel class. So the guessed logic does not play well.

Reply:
so yes, this looks like a bug in master

Reply:
gotcha, i'll open up an issues over in master and see about a proper fix over there